samples/maven-parent/pom.xml

```diff
@@ -28,11 +28,17 @@
 		<wavefront.version>2.2.0</wavefront.version>
 		<spring-cloud.version>2021.0.1</spring-cloud.version>
 		<native.buildtools.version>0.9.10</native.buildtools.version>
+		<maven-compiler-plugin.version>3.10.1</maven-compiler-plugin.version>
```

Comment:
I would not do that. This is managing by Spring Boot already so we should use whatever it provides.

Reply:
Yes, you're absolutely right. I didn't realize that the starter parents inherit from the spring-boot-starter. I will remove the `pluginManagement` and the version override property.